Makefile

```diff
@@ -0,0 +1,5 @@
+lint:
+	swiftlint --fix
```

Comment:
What if we were to have a `lint` and a `lint-fix` or something like that?

Reply:
Thought about that, shouldn't be an issue.